docs/tempo/website/metrics-generator/span_metrics.md

```diff
@@ -1,10 +1,10 @@
 ---
 aliases:
 - /docs/tempo/latest/server_side_metrics/span_metrics/
-title: Span metrics
+title: Generate metrics from spans
 ---
 
-# Generating metrics from spans
+# Generate metrics from spans
 
 The span metrics processor generates metrics from ingested tracing data automatically.
 Span metrics aggregates request, error and duration (RED) metrics from span data.
```

Comment:
Unrelated to this PR specifically, but there's some inconsistency in the use of "span data" and "tracing data" in these two sentences. Might be worth ironing those out.

Reply:
Ursula is currently creating a glossary. I'll ask her to add span data and tracing data.